xml/admin_ceph_datamgm.xml

```diff
@@ -1279,13 +1287,22 @@ rule example2 {
 
   <sect2 xml:id="op-pg-set">
    <title>Setting the number of placement groups</title>
+   <note>
+    <para>
+     Since &ceph; Nautilus (v14.x), you can use the &mgr;
+     <literal>pg_autoscaler</literal> module to auto-scale the PGs as needed.
+     If you want to enable this feature, please refer to
+     <xref linkend="default-pg-and-pgp-counts"/>.
+    </para>
+   </note>
    <para>
-    To set the number of placement groups in a pool, you need to specify the
-    number of placement groups at the time you create the pool (see
-    <xref linkend="ceph-pools-operate-add-pool"/>). Once you have set placement
-    groups for a pool, you may increase the number of placement groups, but you
-    cannot decrease them. To increase the number of placement groups, run the
-    following:
+    If you still need to specify the number of placement groups in a pool
+    manually, you need to specify them at the time of pool creation (see
+    <xref
+    linkend="ceph-pools-operate-add-pool"/>). Once you have set
+    placement groups for a pool, you may increase the number of placement
+    groups, but you cannot decrease them. To increase the number of placement
+    groups, run the following:
```

Comment:
This is no longer true. Since iirc `nautilus` ceph can merge PGs, aka reduce a pg_num. Feel free to just remove these last two sentences.